cg-spec/editors_draft.html

```diff
@@ -2477,9 +2493,10 @@ <h3>Operator Mappings</h3>
 
       <section id="triple-term-comparison">
         <h3>Triple terms with ORDER BY</h3>
-
-        <p>There is an added item to the ordering of terms when they would not 
-          otherwise be ordered. RDF-star triple terms are placed last (highest).</p>
+        <p>
+          There is an added item to the ordering of terms when they would not 
+          otherwise be ordered. RDF-star triple terms are placed last (highest).
```

Comment:
```suggestion
          There is an item added to the ordering of terms when they would not 
          otherwise be ordered: RDF-star triple terms are placed last (highest).
```